packages/lit-html/src/test/lit-html_test.ts

```diff
@@ -34,6 +34,9 @@ import './polyfills.js';
 
 const ua = window.navigator.userAgent;
 const isIe = ua.indexOf('Trident/') > 0;
+// We don't have direct access to DEV_MODE, but this is a good enough
+// proxy.
+const DEV_MODE = render.setSanitizer != null;
```

Comment:
with the replacement rules we have, shouldn't we be able to do the same as in the main source: `const DEV_MODE = true;`?

Reply:
Do we build our tests? It looks like no, making that change causes the prod tests to fail

Reply:
Ah, ok